src/style.css

```diff
@@ -1 +1,68 @@
-/* Styles go here */
+* {
+  box-sizing: border-box;
+  padding: 0;
+  margin: 0;
+}
+
+.logo {
+  cursor: pointer;
+  margin-right: auto;
+  margin-left: 50px;
+}
+
+.header {
+  background: #fff;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+  display: flex;
+  align-items: center;
+}
+
+.nav {
+  margin-right: 50px;
+}
```

Comment:
this margin relates more to the position of the nav inside of the header rather than to the nav itself. just give horizontal paddings for the header and you will achieve the same result. because atm the gap between the sides of the page and the header are located in two different classes